vkubelet/vkubelet.go

```diff
@@ -59,6 +57,7 @@ func New(ctx context.Context, cfg Config) (s *Server, retErr error) {
 		k8sClient:       cfg.Client,
 		resourceManager: cfg.ResourceManager,
 		provider:        cfg.Provider,
+		podCh:           make(chan *corev1.Pod, 1024),
```

Comment:
This seems like a pretty high number.
I'm wondering if the channel is really neccessary?

Reply:
Is the worry here that VK can block up k8s processing?

Reply:
I wouldn't worry about this blocking VK because it will block eventually anyway, be it slow to queue or slow to process.